rero_ils/modules/local_entities/extensions/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Document record extensions."""
+from .authorized_access_point import AuthorizedAccessPointExtension
+
+__all__ = (
+    'AuthorizedAccessPointExtension',
+)
```

Comment:
Did you try to write in "Java style" ^^
I think we could use only a single "extensions.py" file

Reply:
In future, i think we will use more than one extension